src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -1751,6 +1751,26 @@ public function testCopyShouldKeepExecutionPermission()
         $this->assertFilePermissions(767, $targetFilePath);
     }
 
+    public function testDumpToProtectedDirectory()
+    {
+        if (PHP_OS_FAMILY !== 'Windows') {
+            $this->markTestSkipped('This test is specific to Windows.');
+        }
+
+        if (($userProfilePath = getenv('USERPROFILE')) === false || !is_dir($userProfilePath)) {
+            throw new \RuntimeException('Failed to retrieve user profile path.');
+        }
+
+        // todo: is this path valid for non-english installations?
```

Comment:
should be removed, we don't need to answer this until someone reports there is an issue here